src/line_break.c

```diff
@@ -0,0 +1,279 @@
+#include "global.h"
+#include "line_break.h"
+#include "text.h"
+#include "malloc.h"
+
+void StripLineBreaks(u8 *src)
+{
+    u32 currIndex = 0;
+    while (src[currIndex] != EOS)
+    {
+        if (src[currIndex] == 0xFA || src[currIndex] == 0xFE)
+            src[currIndex] = 0;
+        currIndex++;
+    }
+}
+
+void BreakStringKnuth(u8 *src, u32 maxWidth, u32 screenLines, u8 fontId)
```

Comment:
Maybe we could call these "PseudoKnuth" in the meantime.

Reply:
Maybe just remove the mention of Knuth completely, since if it gets updated in the future to use Knuth-Plass it'll be inaccurate then and changing it would be invasive.